test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -855,9 +855,9 @@ def revert_server(self, server, force=False):
         server_stat = server.status(SERVER)[0]
         current_user = pwd.getpwuid(os.getuid())[0]
         try:
-            # remove current user's entry from managers list (if any)
-            a = {ATTR_managers: (DECR, current_user + '@*')}
-            server.manager(MGR_CMD_SET, SERVER, a, sudo=True)
+            # Unset managers list
+            server.manager(MGR_CMD_UNSET, SERVER, 'managers', sudo=True,
+                           expect=True)
```

Comment:
Is it wise to except pass here?  If the manager fails, then you won't have unset the attribute.  Things won't be completely back to defaults at the beginning of the next test.  Maybe at least log something.